dashboard/src/components/InstanceListView/index.tsx

```diff
@@ -20,7 +20,10 @@ import { InstanceCardList } from "./InstanceCardList";
 
 export interface InstanceListViewProps {
   brokers: IServiceBroker[];
-  classes: IClusterServiceClass[];
+  classes: {
```

Comment:
instead of repeating it, we can use `IServiceCatalogState["classes"]` as the type